src/app/api/v1/share-links/[id]/qrcode/route.ts

```diff
@@ -0,0 +1,48 @@
+import { NextResponse } from 'next/server';
+import QRCode from 'qrcode';
+
+import {
+  INVALID_SHLINK_CREDS,
+  NOT_FOUND,
+  UNAUTHORIZED_REQUEST,
+} from '@/app/constants/http-constants';
+import { handleApiValidationError } from '@/app/utils/error-handler';
+import { container, SHLinkEndpointRepositoryToken, SHLinkRepositoryToken } from '@/container';
+import { SHLinkQRCodeRequestDto } from '@/domain/dtos/shlink-qrcode';
+import { ISHLinkRepository } from '@/infrastructure/repositories/interfaces/shlink-repository';
+import { encodeSHLink } from '@/mappers/shlink-mapper';
+import { getSingleSHLinkUseCase } from '@/usecases/shlinks/get-single-shlink';
+
+const shlinkRepo = container.get<ISHLinkRepository>(SHLinkRepositoryToken);
+
+export async function GET(request: Request, { params }: {params: { id: string } }) {
```

Comment:
Please refer to the swagger-docs PR on how to add swagger docs comments to you route. Just the comments nothing extra